jetstream/core/orchestrator.py

```diff
@@ -248,10 +244,9 @@ def __init__(
     # Stage 1
     # At first, a request is placed here in order to get prefilled.
     self._prefill_backlog = queue.Queue()
-    self._prefill_backlog_size_metric = prometheus_client.Gauge(
-        f"jetstream_prefill_backlog_size_{shortuuid.uuid()}",
-        "Size of prefill queue",
-    )
+    jetstream_metrics.prefill_backlog.labels(
```

Comment:
Could you add flag guard to the metric collection for all places? The flag should be consistent with the one in `server_lib.py`.